Fly_Swatter/fire_mode.py

```diff
@@ -58,28 +64,28 @@ def track_lock(realism = 0, projectile_type = "bullet", target_course = "straigh
 
   # once firing solution is calculated, fire, and reset
 
-class azimuth_control():
-  # class for controlling the orientation of the launch system and radar
-  # need to implement controls for slaving launch system to radar
-  def __init__(self, name, status, theta, phi, x, y, z):
-    self.name = name
-    self.status = status
-    self.theta = theta
-    self.phi = phi
-    self.x = x
-    self.y = y
-    self.z = z 
+# class azimuth_control():
```

Comment:
this will be implemented in a later PR where mechanical control of a hypothetical system is created